src/redux/package.js

```diff
@@ -5,7 +5,7 @@ import {
   createRequestReducer,
   createRequestEpic
 } from './request';
-import formatContentType from './utilities';
+import { formatContentType } from './utilities';
```

Comment:
Same thing here. Seems like a view concern.

Reply:
Will move to mod-kb-ebsco very soon.